apps/music/index.html

```diff
@@ -18,7 +18,8 @@
   <link rel="stylesheet" href="/css/app.css">
   <!-- Deferred Scripts -->
   <script defer src="/components/bridge/bridge.js"></script>
-  <script defer src="/shared/js/intl/l20n.js" type="application/javascript;version=1.8"></script>
+  <script defer src="/shared/js/intl/l20n-service.js" type="application/javascript;version=1.8"></script>
+  <script defer src="/shared/js/intl/l20n-client.js" type="application/javascript;version=1.8"></script>
```

Comment:
There very little common code.  In fact, it's just these 25 lines: https://github.com/l20n/l20n.js/blob/962b9a4b82b91a25de63472b2fb783d34355707b/src/bindings/html/shims.js. I think I would prefer to keep the fies separate, at least until we need more shared code;  it makes it easier to debug l20n and see which file the errors are coming from.